openupgrade_scripts/apriori.py

```diff
@@ -8,8 +8,12 @@
     "note": "project_todo",
     "website_sale_delivery_mondialrelay": "website_sale_mondialrelay",
     # odoo/enterprise
+    # OCA/l10n-germany
+    "l10n_de_skr03_mis_reports": "l10n_de_mis_reports",
```

Comment:
This have to be in merged modules, not in renamed ones, as you can't rename 2 modules into one.